src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1102,7 +1101,15 @@ public IRequester getRequesterBuildingForPosition(@NotNull final BlockPos pos)
     @NotNull
     public List<EntityPlayer> getMessageEntityPlayers()
     {
-        return ServerUtils.getPlayersFromUUID(this.world, this.getPermissions().getMessagePlayers());
+        return new ArrayList<>(packageManager.getSubscribers());
+    }
+
+    // TODO use important message for various messages.
+    @Override
+    @NotNull
+    public List<EntityPlayer> getImportantMessageEntityPlayers()
+    {
+        return new ArrayList<>(packageManager.getOfficerSubscribers());
```

Comment:
I just remembered, there is a permission setting in the colony which defines if a rank is a subscriber to the messages. Maybe we should use that?

Reply:
hm we could do that, but this list is also receiving the messages when they're not close to the colony. Not sure if we'd want that for the friends rank

Reply:
Hmm, maybe we make this a separate setting and add it to the list, just to make sure. Especially in the future we'll add a way to add custom ranks to the colony, so people will definitely use this then as well.